content/tutorials/inspecting-infrastructure/index.md

```diff
@@ -0,0 +1,138 @@
+prereqs:
+    - A [Pulumi account](https://app.pulumi.com) and [access token](/docs/pulumi-cloud/access-management/access-tokens/)
+    - The [Pulumi CLI](https://www.pulumi.com/docs/install/)
+    - An [Amazon Web Services](https://aws.amazon.com/) account
+    - The [AWS CLI](https://aws.amazon.com/cli/)
+    - Your desired [language runtime installed](/docs/clouds/aws/get-started/begin/#install-language-runtime)
+
+# The estimated time, in minutes, for new users to complete the topic.
+estimated_time: 10
+
+# # An optional list of collections this tutorial should be belong to. Collections are defined in data/tutorials/collections.yaml.
+# collections:
+#     - some-non-existent-collection
+---
+
+## [Optional] Create a new project
+
+{{< notes type="info" >}}
+
+The commands found in this tutorial can be run against any projects or stacks that you may have already created. Feel free to skip this step and use your own project/stack if that is the case.
+
+{{< /notes >}}
+
+To start, login to the [Pulumi CLI](/docs/cli/commands/pulumi_login/) and ensure it is [configured to use your AWS account](/docs/clouds/aws/get-started/begin/#configure-pulumi-to-access-your-aws-account). Next, [create a new project](/docs/clouds/aws/get-started/create-project/) and replace the default program code with the following:
+
+{{< example-program path="aws-s3bucket-bucketpolicy" >}}
+
+This code example creates the following resources:
+
+- An S3 bucket
+- An S3 bucket policy definition
+- An S3 bucket policy attachment
+
+It also includes two exports that will output the name and ARN of the S3 bucket.
+
+Now run the `pulumi up` command to deploy your resources before moving onto the next steps.
+
+## Inspect your infrastructure
+
+In this section, you will run a number of commands in the Pulumi CLI that will enable you to view more details about the resources you have deployed.
+
+### pulumi stack
+
+The [pulumi stack](/docs/cli/commands/pulumi_stack/) command is used to provide a quick overview of the current stack's status and configuration. Running this command will list the management details, resources, and [output](/docs/concepts/inputs-outputs/#outputs) names and values of the current stack.
+
+Run the `pulumi stack` command as shown below:
+
+{{< video title="Running the pulumi stack command" src="https://www.pulumi.com/uploads/aws-cli-pulumi-stack.mp4" autoplay="true" loop="true" >}}
+
+### pulumi stack graph
+
+The [pulumi stack graph <filename>](/docs/cli/commands/pulumi_stack_graph/) command is used to visualize the dependency graph of a Pulumi stack. This graphical representation can help users to understand the relationships and dependencies between resources in their infrastructure.
+
+Run the `pulumi stack graph <filename>` command as shown below, making sure to replace `<filename>` with the name of the file that you want the graph to be exported to:
```

Comment:
I would mention that the output file is in Graphviz DOT format and offer a link to an online viewer (e.g. https://dreampuf.github.io/GraphvizOnline)